src/components/List/AddCard/AddCardForm/AddCardForm.js

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useRef, useState } from "react";
```

Comment:
This component shows the text area and Add Card green button when you've pressed the Add Card at the bottom of the list